docs/AllRectorsOverview.md

```diff
@@ -5,7 +5,12 @@
 Handles method calls in child of Doctrine EntityRepository and moves them to "$this->repository" property.
 
 ```diff
-{
+ <?php
+
+ use Doctrine\ORM\EntityRepository;
+
+ class SomeRepository extends EntityRepository
+ {
```

Comment:
I also had to create own Formatter factory, so it displays the rest of the code, not only the diff and 3 lines around it.